web/client/epics/annotations.js

```diff
@@ -238,7 +238,7 @@ module.exports = (viewer) => ({
                 const feature = state.annotations.editing;
                 const drawing = state.annotations.drawing;
                 const type = state.annotations.featureType;
-                const multiGeom = state.annotations.config.multiGeometry;
+                const multiGeom = get(state, 'annotations.config.multiGeometry');
```

Comment:
Hi, please use a selector here (create it and test if necessary), since it not the first time you have to do this fix

Also, please check if this fix is needed in other epics